cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/MetadataServiceMain.java

```diff
@@ -113,7 +118,15 @@ protected void addServices(Injector injector, List<? super Service> services,
       EnvironmentOptions options) {
     services.add(injector.getInstance(MetadataService.class));
     services.add(injector.getInstance(MetadataSubscriberService.class));
-    services.add(injector.getInstance(MetadataConsumerSubscriberService.class));
+
+    if (cConf != null && cConf.getStringCollection(
+        Constants.MetadataConsumer.METADATA_CONSUMER_EXTENSIONS_ENABLED_LIST).isEmpty()) {
+      LOG.debug("Skipping enabling MetadataConsumerSubscriberService, "
```

Comment:
Make it info. It's one line on the start of the service and it's important

Reply:
Ack, done.